### PR TITLE
Organ fixes

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl: Fethas
bugfix: the nest buckle now looks for the right organ path
bugfix: Fixed some sprites.
bugfix: No more putting no drops in rechargers,
/:cl:

### CODE REVIEW

**Comment by Tastyfish** [comment]
Fixes #3788

File: code/game/machinery/recharger.dm [hunk 1]
```diff
@@ -16,6 +16,9 @@
 /obj/machinery/recharger/attackby(obj/item/weapon/G as obj, mob/user as mob, params)
 	if(istype(user,/mob/living/silicon))
 		return
+
+	if(G.flags & NODROP)
+		user << "<span class='notice'>you can drop it to charge it!</span>"
```
Comment (Fox-McCloud) [inline code/game/machinery/recharger.dm]: I believe you could put

```			if(!user.drop_item())
				return```

on line 47

File: code/modules/surgery/organs/subtypes/xenos.dm [hunk 2]
```diff
@@ -98,6 +98,8 @@
 			owner.adjustFireLoss(-heal_amt)
 			owner.adjustOxyLoss(-heal_amt)
 			owner.adjustCloneLoss(-heal_amt)
+	else//still should regen some plasma
+		owner.adjustPlasma(plasma_rate-5)//but at a lower rate
```
Comment (Fox-McCloud) [inline code/modules/surgery/organs/subtypes/xenos.dm]: This isn't a bugfix; this is a balance change.

I looked through our code prior to this, and Xenos never regenned plasma when not on weeds---with one sole exception: Hunters would regen 5 plasma every life process if they weren't running.

**Comment by Tastyfish** [comment]
Small note, I believe the /:cl: needs to be on its own line.

**Pull request merged**